PShape_Demo.py

```diff
@@ -0,0 +1,29 @@
+from p5 import *
```

Comment:
We already have a repository for examples! This should ideally go there: https://github.com/p5py/p5-examples